### PR TITLE
Fix minor accessibility issues found in recent scan

### PR DESCRIPTION
**Pull request by scottbommarito**
These changes fix the following accessibility issues:

https://devdiv.visualstudio.com/DevDiv/_workitems/edit/904918
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/904930
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/904941
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/905541
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/910823
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/910473
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/910459
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/905573
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/905742
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/904912
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/917909

image

image

### CODE REVIEW

File: src/NuGetGallery/Views/Pages/Home.cshtml [hunk 1]
```diff
@@ -64,7 +64,7 @@
 
     <div class="container text-center what-is-nuget">
         <div class="row">
-            <h1>What is NuGet?</h1>
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Pages/Home.cshtml]: What's the difference in look?

File: src/NuGetGallery/Views/Organizations/Add.cshtml [hunk 2]
```diff
@@ -26,9 +26,9 @@
             <div>
                 <aside class="col-sm-3 col-sm-push-9">
                     @Html.Label("Logo")
-                    <a href="#" id="transform-tooltip" data-toggle="tooltip" title="We use the email address for your organization and gravatar.com to get the logo.">
+                    <span class="btn-link" id="transform-tooltip" data-toggle="tooltip" data-placement="right" title="We use the email address for your organization and gravatar.com to get the logo.">
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Organizations/Add.cshtml]: What is this fixing?

**Comment by loic-sharma** [comment]
Could you add screenshots for the things that have changed (default input color, heading size reduction, etc..)

File: src/NuGetGallery/Views/Pages/Home.cshtml (hunk 1 above)
Comment (scottbommarito) [reply src/NuGetGallery/Views/Pages/Home.cshtml]: Very minor, will grab a pic soon. I was considering updating the CSS to compensate for this change but I don't think it's necessary given how minor the change is.

image

**Comment by scottbommarito** [comment]
@loic-sharma - done. IMO it's pretty difficult to see the difference.

File: src/NuGetGallery/Views/Organizations/Add.cshtml (hunk 2 above)
Comment (scottbommarito) [reply src/NuGetGallery/Views/Organizations/Add.cshtml]: The changes to this file are for

https://devdiv.visualstudio.com/DevDiv/_workitems/edit/905541
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/904941
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/904930
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/904918

Basically the `<a>` tag showed up in the tab order as a link, which was confusing to screen readers. By changing it to a `<span>`, it no longer does and is less confusing. I also fixed it--it was intended to be a Bootstrap tooltip but was not properly configured with `.tooltip()`.

Comment (joelverhagen) [reply src/NuGetGallery/Views/Organizations/Add.cshtml]: Let's wait on this change/PR until we know the right behavior for tooltips + screenreader.

Comment (scottbommarito) [reply src/NuGetGallery/Views/Organizations/Add.cshtml]: Looked into it some more and this change was incorrect. After trying to get the tooltip working properly, I decided that changing the tooltip into text beneath the image is both better UI and more accessible. Thoughts?

image

**Pull request merged**